core/src/main/java/org/infinispan/factories/threads/DefaultThreadFactory.java

```diff
@@ -13,6 +13,7 @@
 
    public static final String DEFAULT_PATTERN = "%c-%n-p%f-t%t";
 
+   private final String name;
```

Comment:
Seems this is never used but in the getter.  Did we want to add this to the name of the threads when they are created?